dbt/models/iasworld/schema/iasworld.owndat.yml

```diff
@@ -250,6 +261,12 @@ sources:
                 - user27
               allow_interior_space: true
               config: *unique-conditions
+              meta:
+                description: >
+                  The following columns should have no leading or trailing
+                  whitespace: own1, own2, addr1, addr2, addr3, addr4, addr5,
+                  adrpre, adrdir, adrstr, adrsuf, unitdesc, unitno, cityname,
+                  statecode, zip1, zip2, user27
```

Comment:
Same here RE: whitespace tests being hard to debug. If we keep them, we should probably split these out into separate tests for every field.